boards/arm/control_freak/control_freak_defconfig

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2025 RIT Launch Initiative
+# SPDX-License-Identifier: Apache-2.0
+
+# System
+CONFIG_CLOCK_CONTROL=y
+CONFIG_ARM_MPU=y
+CONFIG_HW_STACK_PROTECTION=y
+
+CONFIG_UART_CONSOLE=y
+CONFIG_CONSOLE=y
+
+CONFIG_CLOCK_CONTROL=y
+
+# IO
+CONFIG_GPIO=y
+CONFIG_PINCTRL=y
+CONFIG_SERIAL=y
+CONFIG_SPI=y
+CONFIG_SPI_ASYNC=y
+CONFIG_LED=y
+
+
+
+CONFIG_DEBUG=y
```

Comment:
Lets move this to a debug config? I would avoid having unnecessary configs in the board level config. We should also preferably run non debug modes in actual flight

Reply:
oops